pkg/wallet/account.go

```diff
@@ -55,6 +57,12 @@ type Contract struct {
 
 	// Indicates whether the contract has been deployed to the blockchain.
 	Deployed bool `json:"deployed"`
+
+	// InvocationScript returns invocation script for deployed contracts.
+	// In case contract is not deployed or has 0 arguments, this field is ignored.
+	// It might be executed on a partially formed tx, and is primarily needed to properly
+	// calculate network fee for complex contract signers.
+	InvocationScript func(tx *transaction.Transaction) ([]byte, error) `json:"-"`
```

Comment:
Why not `InvocationBuilder` of some kind? I fear this is gonna be confused with transaction/block/payload invocation scripts that are static data.

Reply:
Bugged me too, not strong in naming, will fix.

Reply:
Fixed, also rebased on master.